desafio-8/desafio-8.css

```diff
@@ -0,0 +1,80 @@
+header {
+ padding: 20px 0;
+ background-color: #9932CC;
+ color: #ffffff;
+ display: flex;
+ align-items: center;
+ justify-content: center;
+ flex-direction: row;
+}
+
+body{
+ background-color:#222222;
+ font-family: Inter;
+}
+
+main {
+ width: 940px;
+ margin: 0 auto;
+}
+
+form {
+ margin: 40px 0;
+ display: flex;
+ justify-content: center;
+ flex-direction: column;
+ align-items: center;
+ color: #ffffff;
+ gap: 20px;
+}
+
+.input-padrao {
+ margin: 0 0 20px;
+ padding: 10px 25px;
+ width: 60%;
+ border: 0;
+ border-bottom: 2px solid gray;
+ font-size: 10px;
+ color: white;
+ padding: 7px 0;
```

Comment:
this input has two property padding:
 padding: 10px 25px;

and this padding,  padding: 7px 0;

Reply:
Really! I changed for one padding